build-index.sh

```diff
@@ -168,6 +174,16 @@ function list_dir() {
         fi
 	      diff=$(rfcdiff "https://tools.ietf.org/id/${file}.txt" "$this_githubio")
         td "$(a "$diff" 'diff with last submission' diff "$file")"
+        if [[ "${#files[@]}" -eq 1 ]]; then
+            td ""
+        else
+            this_issue_label=$(./lib/extract-frontmatter.py ${file}.md github-issue-label)
```

Comment:
```suggestion
            this_issue_label=$(./lib/extract-frontmatter.py $(ls ${file}.{md,xml}) github-issue-label)
```

Or something like that; though fixing this up is probably something I can sort out.

Reply:
Hmm. Either that or move the logic into the script (i.e., just hand it the base name). Preferences?